CHANGES.rst

```diff
@@ -1,8 +1,13 @@
-Version 1.0.3
+Version 1.0.4
```

Comment:
1.0.4 will be added while creating a new release. Remove this and put unreleased in 1.0.3